### PR TITLE
dev/core#2715 [REF] Move another 2 functions to the financial processor class

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
dev/core#2715 [REF] Move another 2 functions to the financial processor class

Before
----------------------------------------
`createFinancialItemsForLine` & `getMultiplier` are part of the borg

After
----------------------------------------
They are on the more focussed financialProcessor class

Technical Details
----------------------------------------
The process involves
- checking the selfs
- temporarily making public
- making private again once the calling function is also moved

Comments
----------------------------------------

### CODE REVIEW

File: CRM/Contribute/BAO/Contribution.php [hunk 1]
```diff
@@ -3694,11 +3628,10 @@ public static function recordFinancialAccounts(&$params) {
    *
    */
   public static function updateFinancialAccounts(&$params, $context = NULL) {
-    $trxnID = NULL;
```
Comment (eileenmcnaughton) [inline CRM/Contribute/BAO/Contribution.php:3697]: unused

**Comment by eileenmcnaughton** [comment]
@colemanw are you OK to merge this?

**Comment by eileenmcnaughton** [comment]
@colemanw @demeritcowboy anyone ok to merge this - it's just moving functions around

**Comment by demeritcowboy** [comment]
I'm not sure how to run this but the code looks ok. Neither function seems called during a backend contribution create/edit.

I can't see why getMultiplier needs to be made public but ok.

**Comment by eileenmcnaughton** [comment]
@demeritcowboy these functions are covered by around about 1000 tests ::-) All the ones that deal with money

I can't remember re getMultiplier - it's been a while but basically the idea is to get all the functions relevant to the financial processing workflow to that class & then make them play nice

**Comment by demeritcowboy** [comment]
Ok I see e.g. if you change the financial type on a contribution it gets called.

**Comment by eileenmcnaughton** [comment]
Thanks @demeritcowboy

**Pull request merged**